bin/daps-init

```diff
@@ -189,6 +183,13 @@ while true ; do
     esac
 done
 
+if [[ -z "$DOC_DIR" ]]; then
+  exit_on_error "Please specify a directory"
```

Comment:
I suggest to change this to "Please specify a directory with --docdir" to make it clear where the input is expected.